apps/emqx/src/emqx_persistent_session_ds/ChannelInfo.asn

```diff
@@ -0,0 +1,55 @@
+ChannelInfo DEFINITIONS ::=
+BEGIN
+  OtherProtocol ::= SEQUENCE {
+    proto [0] OCTET STRING,
+    ver [1] INTEGER
+  }
+
+  ProtoVer ::= CHOICE {
+    mqtt [0] INTEGER(0..255),
+    mqtt-sn [1] INTEGER(0..255),
+    coap [2] INTEGER(0..255),
+    other [3] OtherProtocol
+  }
+
+  ChannelInfo ::= SEQUENCE {
+    -- Stats:
```

Comment:
there is maybe no need to keep the stats in ds ?

Reply:
Last time QA raised an issue about lack of stats for disconnected durable sessions, so we added that feature.